usaspending_api/transactions/models/source_procurement_transaction.py

```diff
@@ -328,6 +328,10 @@ class SourceProcurementTransaction(models.Model):
     veterinary_hospital = models.BooleanField(null=True, blank=True)
     woman_owned_business = models.BooleanField(null=True, blank=True)
     women_owned_small_business = models.BooleanField(null=True, blank=True)
+    ser_disabvet_own_bus_join_ven = models.BooleanField(null=True, blank=True)
```

Comment:
Can you place the new columns in their respective alphabetical order? While we don't carry this pattern forward across all tables, it would be good to carry it forward here given the fields are currently in order.